p2p/module_raintree_test.go

```diff
@@ -15,12 +15,14 @@ import (
 	"testing"
```

Comment:
```golang
for i := 0; i < numAddressessToBeAdded; i++ {
				newAddr, _ := crypto.GenerateAddress()
				network.AddPeerToAddrBook(&types.NetworkPeer{Address: newAddr})
			}
```

Catch the error in `newAddr, _ := crypto.GenerateAddress()`

Reply:
Addressed here

https://github.com/pokt-network/pocket/pull/235/files#diff-2a2651c6ad9e1f7d34a394f137a8da69f8204d60e0d22ba7b304cb48d72e5d77R118